### PR TITLE
DAOS-8589 test: Fix container/query_attribute.py in weekly test

### PR DESCRIPTION
**Pull request by shimizukko**
Use TestContainer to create a container.
Refactored the test and the tag.

Skip-unit-tests: true
Test-tag: cont_query_attr cont_list_attrs
Signed-off-by: Makito Kano <makito.kano@intel.com>

### CODE REVIEW

File: src/tests/ftest/container/query_attribute.py [hunk 1]
```diff
@@ -55,21 +46,27 @@ def test_container_query_attr(self):
         Use Cases:
             Test container query, set-attr, get-attr, and list-attrs.
 
-        :avocado: tags=all,pool,small,full_regression,cont_query_attr
+        :avocado: tags=all,full_regression
+        :avocado: tags=small
+        :avocado: tags=container,cont_query_attr
```
Comment (daltonbohning) [inline src/tests/ftest/container/query_attribute.py:51]: Was `pool` intentionally removed?

File: src/tests/ftest/container/query_attribute.py [hunk 2]
```diff
@@ -156,22 +157,33 @@ def test_list_attrs_long(self):
         Use Cases:
             Test daos container list-attrs with 50 attributes.
 
-        :avocado: tags=all,pool,small,full_regression,cont_list_attrs
+        :avocado: tags=all,full_regression
+        :avocado: tags=small
+        :avocado: tags=container,cont_list_attrs
```
Comment (daltonbohning) [inline src/tests/ftest/container/query_attribute.py:162]: Was `pool` intentionally removed?

File: src/tests/ftest/container/query_attribute.py [hunk 3]
```diff
@@ -55,21 +46,27 @@ def test_container_query_attr(self):
...
+        :avocado: tags=small
+        :avocado: tags=container,cont_query_attr
         """
-        # Test pool query.
-        self.create_pool_container()
+        # Create a pool and a container.
+        self.add_pool()
+        self.add_container(pool=self.pool)
+
+        self.daos_cmd = DaosCommand(self.bin)
```
Comment (daltonbohning) [inline src/tests/ftest/container/query_attribute.py:57]: I think we can do this like in `test_list_attrs_long` below
```suggestion
        self.daos_cmd = self.get_daos_command()
```

**Review by daltonbohning — APPROVED** [review]

**Review by dinghwah — DISMISSED** [review]
LGTM

**Comment by sylviachanoiyee** [comment]
The changes in daos_utils only impacts the query_attribute test?
Can we run all tests to make sure nothing broke?

**Review by daltonbohning — APPROVED** [review]

**Review by dinghwah — APPROVED** [review]
LGTM

**Comment by shimizukko** [comment]
The refactoring in daos_utils.py was irrelevant to this PR, so reverted it to make the PR smaller.

**Pull request merged**